spec/features/super_admin/logging/view_logs_for_ip_spec.rb

```diff
@@ -0,0 +1,78 @@
+describe 'View authentication requests for an IP', type: :feature do
+  let(:user) { create(:user, organisation: user_organisation) }
+  let(:user_organisation) { create(:organisation) }
+  let(:user_location) { create(:location, organisation: user.organisation) }
+  let(:ip_1) { create(:ip, location_id: user_location.id, address: '1.2.3.4') }
+
+  let(:user_2) { create(:user, organisation: user_2_organisation) }
+  let(:user_2_organisation) { create(:organisation) }
+  let(:user_2_location) { create(:location, organisation: user_2.organisation) }
```

Comment:
Why not just `user_2`, `location_2`, `organisation_2`? Simpler and more understandable

Reply:
Also `let(:user_2_organisation) { create(:organisation) }` not required